autoscale_cloudcafe/autoscale_fixtures/config.py

```diff
@@ -10,6 +10,9 @@ class AutoscaleConfig(ConfigSectionInterface):
     """
     SECTION_NAME = 'autoscale'
 
+    lc_image_ref = None
```

Comment:
Mutable staaaaate :-(

I do like the fact that this makes the data structure more regular-python-ish and less of a wrapper around a weird dictionary.  But I don't like the fact that this change results in even less documentation for these two properties.  It took me a minute to understand what they were supposed to be, and why this was in a "config section" at all.  Can you add some `:ivar:` documentation to the class docstring?

Also, these are default-NULL, which I am coming to believe is always an antipattern.  I wrote some about this in Python: https://glyph.twistedmatrix.com/2015/09/python-option-types.html and of course there's the option of `sumtypes`.  When and why are these values `None`?